pkg/limits/frontend/frontend_test.go

```diff
@@ -171,10 +177,10 @@ func TestFrontend_ExceedsLimits(t *testing.T) {
 		// and instance 1 for the data for partition 1.
 		expectedStreamUsageRequest: []*logproto.GetStreamUsageRequest{{
 			Tenant:       "test",
-			StreamHashes: []uint64{0x1, 0x2},
+			StreamHashes: []uint64{0x2},
```

Comment:
Here is the main fix. When we stopped sending the partition ID in `GetStreamusageRequest`, we started sending just the stream hashes for the owned partitions, instead of all stream hashes. However, because `expectedStreamUsageRequest` was not being used in the test, this was never asserted. This test should have failed when we made the change a couple weeks ago.